school-closure/school.fred

```diff
@@ -45,13 +46,13 @@ condition SCHOOL {
 
     state CheckLocalEpidemic {
         wait(0)
-	if (local_closure_trigger <= current_count_of_INF.Is_in_School) then next(Close)
+	    if (local_closure_trigger <= current_count_of_INF.Is_in_School) then next(Close)
```

Comment:
The spacing here may still be off?

This would be a subsequent PR, but maybe we should have a script to format the files which just calls John's formatter?